bluesky/run_engine.py

```diff
@@ -111,7 +111,7 @@ class RunEngine:
 
     _loop = loop  # just a convenient way to inspect the global event loop
     state = LoggingPropertyMachine(RunEngineStateMachine)
-    _UNCACHEABLE_COMMANDS = ['pause', 'subscribe', 'unsubscribe']
+    _UNCACHEABLE_COMMANDS = ['pause', 'subscribe', 'unsubscribe', 'stage']
```

Comment:
This fixes one case:

- checkpoint
- stage
- INTERRUPTION

but not another case:

- checkpoint
- stage
- unstage
- INTERRUPTION

This second case affects other messages as well and should be addressed in a future PR. The first case is more common, so I'd like to address it here.